serverless/containers/concepts.mdx

```diff
@@ -127,15 +127,15 @@ Secrets are an extra-secure type of environment variable. They are environment v
 
 ## Serverless
 
-Serverless allows you to deploy your Functions (FaaS) and Containerized Applications (CaaS) in a managed infrastructure. Scaleway ensures the deployment, the availability and the scalability of all your projects.
+Serverless allows you to deploy your Functions (FaaS) and Containerized Applications (CaaS) in a managed infrastructure. Scaleway ensures the deployment, the availability, and the scalability of all your projects.
 
 </Concept>
 
 <Concept>
 
 ## Serverless Framework
 
-Serverless.com (Serverless Framework) is a tool that enable to deploy serverless applications without having to manages Serverless Container's API call. Just write your configuration in a yaml and deploy, it handles everything event image building.
+Serverless.com (Serverless Framework) is a tool that allows you to deploy serverless applications without having to manage Serverless Container’s API call. Simply write and deploy a YAML configuration file, everything else is handled automatically, even the image building.
```

Comment:
je pense qu'on peut éviter ce type de mot tout court hehe
je dirais "Write and deploy..." directement